.env.template

```diff
@@ -4,3 +4,5 @@ S3_KEY =
 S3_SECRET =
 S3_BUCKET_NAME =
 ROLLBAR_ACCESS_TOKEN =
+client_id=
+:client_secret
```

Comment:
when it comes to env we use uppercase names and I would also add github in the name to know later for which api this client and secret is about so `GITHUB_CLIENT_ID=` and `GITHUB_CLIENT_SECRET` or just `GITHUB_SECRET`